src/components/SingleAutocompleteSelectField/SingleAutocompleteSelectField.tsx

```diff
@@ -72,6 +77,7 @@ const SingleAutocompleteSelectFieldComponent: React.FC<SingleAutocompleteSelectF
     onChange,
     onFetchMore,
     FormHelperTextProps,
+    useNakedInput = false,
```

Comment:
this prop sounds too much like a hook, use something like `nakedInput`/`isNakedInput`